1_Get_Started/practice/prototypes_dongbkim.js

```diff
@@ -0,0 +1,65 @@
+function randMax(max) {
+    return Math.trunc(1E9 * Math.random()) % max;
+}
+
+var reel = {
+    symbols: [
+        "♠", "♥", "♦", "♣", "☺", "★", "☾", "☀"
+    ],
+    spin() {
+        if (this.position == null) {
+            this.position = randMax(
+                this.symbols.length - 1
+            );
+        }
+        this.position = (
+            this.position + 100 + randMax(100)
+        ) % this.symbols.length;
+    },
+    display() {
+        if (this.position == null) {
+            this.position = randMax(
+                this.symbols.length - 1
+            );
+        }
+        return this.symbols[this.position];
+    }
+};
+
+var slotMachine = {
+    reels: [
+		Object.create(reel),
+		Object.create(reel),
+		Object.create(reel),
+    ],
+    spin() {
+        this.reels.forEach(function spinReel(reel){
+            reel.spin();
+        });
+    },
+    display() {
+		let symbol = this.reels[0].symbols;
```

Comment:
함수 내에서 var은 함수 전체 스코프, let은 block스코프이므로 가독성 측면에서 symbol을 var로 바꾸는거 어떤가요? 블록 스코프 내부에서 사용할 것들만 let으로 쓰면 더 가독성이 좋아진다고 생각하는데 어떻게 생각하시나요? 

```suggestion
		var symbol = this.reels[0].symbols;
```

Reply:
호찬님 말에 전적으로 동의합니다. block 내부에서만 쓸 목적으로 let을 사용했는데, symbol을 다른 곳에서도  활용할 부분이 있을까요?

Reply:
넵. 감사합니다.

Reply:
function 전체 범위에 걸쳐 정의된 것이여서 var를 제안한 것이었습니다! 그것이 유일하게 function 전체 범위 내부에서 var을 사용하여 얻을 수 있는 가독성의 이점라고 생각했습니다.